apps/ideaspace/src/components/modules/IdeaspaceNavCard/IdeaspaceNavCard.js

```diff
@@ -20,6 +22,7 @@ const IdeaspaceNavCard = ({
     >
       <CardTitleDiv>{cardTitle}</CardTitleDiv>
       <CardText>{cardDescription}</CardText>
+      <LikeButton>{numberOfLikes}</LikeButton>
```

Comment:
Hey @RETprojects is this where you are having trouble seeing the changes for the likes?